README.md

```
@@ -164,6 +164,13 @@ console.log(`/table/YOUR_TABLE_ID?gameStart=${futureTime}`);
 - ✅ **Graceful fallback** - invalid dates are ignored
 
 
+### Enable Bitcoin Payments
+
+From the .env
+
+* BTC_PAY_SERVER_URL=http://localhost:3001
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)